task/clamav-scan/0.1/clamav-scan.yaml

```diff
@@ -40,7 +42,6 @@ spec:
       computeResources:
         limits:
           memory: 4Gi
-          cpu: 2
         requests:
           memory: 512Mi
           cpu: 10m
```

Comment:
How much CPU does this container use? OCP will only guarantee the CPU that is specified in the request. So currently this container is only guaranteed to get 10m. You need to increase that. I thought you wanted to put 1 whole CPU here.

Reply:
look good now, I cannot resolve this thread, looks like I lack some permission on this repository...

Reply:
Ack, resolving the thread.